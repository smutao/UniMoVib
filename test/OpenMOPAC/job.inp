a test job

 $contrl
   qcprog="mopac"
 $end

 $qcdata
   fchk="CH4.out"
 $end
