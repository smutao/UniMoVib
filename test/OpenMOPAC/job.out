 =======================================================================================================================
       _/     _/                _/     _/            _/     _/
      _/     _/            _/  _/|  _/_/            _/     _/  _/  _/        UniMoVib:          A unified interface for
     _/     _/   _/_/_/       _/ |_/ _/   _/_/_/   _/     _/      _/           molecular harmonic vibrational frequency
    _/     _/  _/    _/  _/  _/  _/ _/  _/    _/  _/     _/  _/  _/_/_/        calculations.
   _/     _/  _/    _/  _/  _/     _/  _/    _/  _/    _/   _/  _/   _/
  _/     _/  _/    _/  _/  _/     _/  _/    _/   _/  _/    _/  _/   _/       Ver. 1.1.1, Jan 14, 2018
   _/_/_/   _/    _/  _/  _/     _/   _/_/_/     _/_/     _/  _/_/_/         https://github.com/zorkzou/UniMoVib
 =======================================================================================================================


 QC Program:      MOPAC
 FCHK file:       CH4.out


 Cartesian coordinates (Angstrom)
 ------------------------------------------------------------------------------------------
   No.   Atom    ZA                 X             Y             Z                Mass
 ------------------------------------------------------------------------------------------
     1    C       6            0.00000000    0.00000000    0.00000000           12.01070000
     2    H       1            1.11160000    0.00000000    0.00000000            1.00794000
     3    H       1           -0.37050000    1.04800000    0.00000000            1.00794000
     4    H       1           -0.37050000   -0.52400000   -0.90760000            1.00794000
     5    H       1           -0.37050000   -0.52400000    0.90760000            1.00794000
 ------------------------------------------------------------------------------------------


 <<< SYMMETRY >>>
 Molecular Point Group (Z)   :   Td  
 Molecular Point Group (Z+M) :   Td  


 ************************************
 ***  Properties of Normal Modes  ***
 ************************************

 Results of vibrations:
 Normal frequencies (cm**-1), reduced masses (AMU), force constants (mDyn/A)

                                                   1                                 2                                 3
          Irreps                                  T2                                T2                                T2
     Frequencies                           1379.4824                         1379.7372                         1379.8141
  Reduced masses                              1.1925                            1.1924                            1.1925
 Force constants                              1.3370                            1.3375                            1.3376
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   6        -0.11786   0.04419   0.03046      -0.04472  -0.04026  -0.11466       0.02965   0.11488  -0.05191
           2   1        -0.04967  -0.20765  -0.14143      -0.01900   0.18896   0.53776       0.01245  -0.53811   0.24501
           3   1         0.41301   0.18224  -0.14367       0.24906   0.07770   0.53535      -0.30664  -0.06441   0.24077
           4   1         0.56191  -0.19010  -0.11638      -0.00876  -0.11789   0.00577      -0.10339  -0.48388   0.33277
           5   1         0.47914  -0.31101   0.03847       0.31164   0.33099   0.28744       0.04421  -0.28250  -0.20002

                                                   4                                 5                                 6
          Irreps                                   E                                 E                                T2
     Frequencies                           1411.3618                         1411.4227                         3102.7211
  Reduced masses                              1.0079                            1.0079                            1.0902
 Force constants                              1.1829                            1.1830                            6.1838
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   6        -0.00021   0.00041  -0.00012       0.00011  -0.00022   0.00036       0.03194   0.03349  -0.07306
           2   1        -0.00012   0.41214  -0.27973       0.00007   0.28132   0.41237      -0.31836   0.01697  -0.03703
           3   1         0.39058   0.13827   0.28084       0.26415   0.09330  -0.41573       0.08930  -0.18989  -0.03703
           4   1         0.03469  -0.43969   0.23974      -0.47033   0.05377   0.16109       0.13214   0.18098   0.24703
           5   1        -0.42269  -0.11559  -0.23943       0.20480  -0.42575  -0.16199      -0.28371  -0.40714   0.69758

                                                   7                                 8                                 9
          Irreps                                  T2                                T2                                A1
     Frequencies                           3102.8695                         3102.9824                         3214.7510
  Reduced masses                              1.0902                            1.0903                            1.0079
 Force constants                              6.1844                            6.1849                            6.1373
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   6        -0.06875   0.05211  -0.00617       0.04164   0.06035   0.04588      -0.00001  -0.00002   0.00010
           2   1         0.68396   0.02640  -0.00309      -0.41413   0.03059   0.02321      -0.49990  -0.00003   0.00001
           3   1         0.21633  -0.68398  -0.00308       0.17098  -0.39342   0.02318       0.16660  -0.47124  -0.00004
           4   1        -0.02302   0.04309   0.02583      -0.25705  -0.36275  -0.65806       0.16648   0.23545   0.40778
           5   1        -0.05809  -0.00650   0.05390       0.00402   0.00644   0.06498       0.16698   0.23607  -0.40894

 Results of translations and rotations:

                                                   1                                 2                                 3
          Irreps                                  T2                                T2                                T2
     Frequencies                              8.6057                            6.4482                           -1.6585
  Reduced masses                              3.2085                            3.2085                            3.2085
 Force constants                              0.0001                            0.0001                           -0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   6         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           2   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           3   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           4   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           5   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721

                                                   4                                 5                                 6
          Irreps                                  T1                                T1                                T1
     Frequencies                             46.9967                           24.4318                          -26.5385
  Reduced masses                              1.0079                            1.0079                            1.0079
 Force constants                              0.0013                            0.0004                           -0.0004
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   6         0.00000  -0.00000  -0.00000       0.00000  -0.00000  -0.00000      -0.00000   0.00000  -0.00000
           2   1         0.00000   0.61238   0.00000       0.00000   0.00000   0.00000      -0.00000  -0.00000   0.61238
           3   1        -0.57735  -0.20411  -0.00000      -0.00000  -0.00000  -0.57735       0.00000   0.00000  -0.20411
           4   1         0.28867  -0.20411  -0.00000       0.00000  -0.50000   0.28867       0.50000   0.00000  -0.20411
           5   1         0.28867  -0.20411  -0.00000       0.00000   0.50000   0.28867      -0.50000   0.00000  -0.20411


 *********************************************
 ***   Thermal Contributions to Energies   ***
 *********************************************

 Molecular mass            :        16.042460    AMU
 Elec. total energy        :         0.000000    Hartree
 Scale factor of Frequency :         1.000000
 Rot. symmetry number      :        12
 The Td   point group is used to calculate rotational entropy.

 Principal axes and moments of inertia in atomic units:
                                     1                   2                   3
     Eigenvalues --                 11.859803           11.859839           11.859873
           X                         0.000000            1.000000            0.000000
           Y                         0.000000            0.000000            1.000000
           Z                        -1.000000            0.000000            0.000000

 Rotational temperatures             7.303141            7.303119            7.303098    Kelvin
 Rot. constants A, B, C              5.075943            5.075928            5.075913    cm**-1
                                   152.172953          152.172481          152.172052    GHz


 #   1    Temperature =       298.15000 Kelvin         Pressure =         1.00000 Atm
 ====================================================================================

 Thermal correction energies                              Hartree            kcal/mol
 Zero-point Energy                          :            0.044390           27.855409
 Thermal correction to Energy               :            0.047261           29.656990
 Thermal correction to Enthalpy             :            0.048206           30.249476
 Thermal correction to Gibbs Free Energy    :            0.027018           16.954003
 ====================================================================================


 UniMoVib job terminated correctly! Sun Jan 14 13:29:11 2018
